src/plugins/search/search.js

```diff
@@ -88,75 +88,70 @@ export function search(query) {
     data = data.concat(Object.keys(INDEXS[key]).map(page => INDEXS[key][page]))
   })
 
-  query = query.trim()
-  let keywords = query.split(/[\s\-，\\/]+/)
-  if (keywords.length !== 1) {
-    keywords = [].concat(query, keywords)
-  }
-
-  for (let i = 0; i < data.length; i++) {
-    const post = data[i]
-    let isMatch = false
-    let resultStr = ''
-    const postTitle = post.title && post.title.trim()
-    const postContent = post.body && post.body.trim()
-    const postUrl = post.slug || ''
-
-    if (postTitle && postContent) {
-      keywords.forEach(keyword => {
-        // From https://github.com/sindresorhus/escape-string-regexp
-        const regEx = new RegExp(
-          keyword.replace(/[|\\{}()[\]^$+*?.]/g, '\\$&'),
-          'gi'
-        )
-        let indexTitle = -1
-        let indexContent = -1
-
-        indexTitle = postTitle && postTitle.search(regEx)
-        indexContent = postContent && postContent.search(regEx)
-
-        if (indexTitle < 0 && indexContent < 0) {
-          isMatch = false
-        } else {
-          isMatch = true
-          if (indexContent < 0) {
-            indexContent = 0
-          }
-
-          let start = 0
-          let end = 0
-
-          start = indexContent < 11 ? 0 : indexContent - 10
-          end = start === 0 ? 70 : indexContent + keyword.length + 60
-
-          if (end > postContent.length) {
-            end = postContent.length
-          }
-
-          const matchContent =
-            '...' +
-            escapeHtml(postContent)
-              .substring(start, end)
-              .replace(regEx, `<em class="search-keyword">${keyword}</em>`) +
-            '...'
-
-          resultStr += matchContent
+  var options = {
+    shouldSort: true,
+    includeMatches: true,
+    threshold: 0.45,
+    location: 0,
+    distance: 100,
+    maxPatternLength: 32,
+    minMatchCharLength: 1,
+    keys: [{
+      name: 'title',
+      weight: 0.75
+    }, {
+      name: 'body',
+      weight: 0.25
+    }]
+  };
+  var fuse = new Fuse(data, options);
```

Comment:
where's Fuse coming from?

Reply:
gotta include it as in index as a separate dependency.

cdn here:
https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.4.4/fuse.min.js

documentation:
https://fusejs.io/

Reply:
could you please update the docs to reflect this within this PR?